doc/api/util.md

```diff
@@ -461,6 +461,7 @@ added: v8.0.0
 -->
 
 * `original` {Function}
+Returns: {Function} a promise style function
```

Comment:
It is somewhat inconsistent throughout this documentation how we do this but most cases use the star to better outline the return value.

Reply:
See the end of the Style Guide: https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md

Reply:
"promise style function" doesn't really work in this regard, either this should be just `{Function}` as the return type or the "original" line should be expanded as well.